.github/workflows/sycl_post_commit.yml

```diff
@@ -9,6 +9,8 @@ on:
     - sycl
     paths:
     - .github/workflows/sycl_post_commit.yml
+    - .github/workflows/sycl_windows_build_and_test.yml
+    - .github/workflows/sycl_macos_build_and_test.yml
```

Comment:
Shouldn't we include all workflow files used by sycl_post_commit recursively? For instance, it's not clear why `sycl_linux_build_and_test.yml` is not in the list.

Reply:
Good point!